### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by Torfeltorfel**
- removed all default styling & example code
- installed styled-component

### CODE REVIEW

File: src/app/App.tsx [hunk 1]
```diff
@@ -1,54 +1,13 @@
...
-            <Route path="/">
-              <main>Home</main>
-            </Route>
-          </Routes>
-        </header>
-      </div>
+      <Routes>
+        <Route path="/about"></Route>
+        <Route path="/"></Route>
+      </Routes>
```
Comment (phlgr) [inline src/app/App.tsx]: Please no empty routes

File: src/app/globals.css [hunk 2]
```diff
@@ -1,13 +1,14 @@
-body {
```
Comment (phlgr) [inline src/app/globals.css:1]: These should be converted to Globalstyles

**Review by phlgr — CHANGES_REQUESTED** [review]

File: src/app/globals.css (hunk 2 above)
Comment (Torfeltorfel) [reply src/app/globals.css:1]: changed and pushed

File: src/app/main.tsx [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './globals.css';
+import GlobalStyles from './GlobalStyles';
```
Comment (phlgr) [inline src/app/main.tsx:3]: You need to use GlobalStyles. Now you are just importing it.

**Review by phlgr — CHANGES_REQUESTED** [review]

File: src/app/main.tsx (hunk 3 above)
Comment (Torfeltorfel) [reply src/app/main.tsx:3]: ahh, I added it to the ReactDOM.render

**Pull request merged**